amplify/src/main/java/com/github/stkent/amplify/tracking/interfaces/IEventBasedRule.java

```diff
@@ -25,6 +25,8 @@
  */
 public interface IEventBasedRule<T> {
 
+    boolean shouldAllowFeedbackPromptByDefault();
```

Comment:
Should also add docs to the other methods in this interface to note that neither should be called unless the cached event value exists (i.e. is not equal to the default tracking value set by the manager, because the event has happened before).